src/formatDate.js

```diff
@@ -8,7 +8,34 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat.slice(3);
+  const newSeparator = toFormat.slice(3);
+  const splitDate = date.split(oldSeparator);
```

Comment:
There's a potential issue with splitting the date using `oldSeparator` directly. Since `oldSeparator` is an array due to the incorrect use of `slice`, `split` may not work as intended. After correcting `oldSeparator` to be a string, this line should work correctly.